aikau/src/main/resources/alfresco/lists/AlfList.js

```diff
@@ -411,10 +422,14 @@ define(["dojo/_base/declare",
             this.processLoadedData(this.currentData);
             this.renderView();
          }
-         else
+         else if (this.loadDataImmediately)
          {
             this.loadData();
          }
+         else
+         {
+            // No action required, wait for either a reload request or data to be published
+         }
```

Comment:
I'm not a fan of empty else blocks - that's how code fails silently. Is it worth at least adding a log message here?